controller/workitem_blackbox_test.go

```diff
@@ -1229,6 +1229,51 @@ func (s *WorkItem2Suite) TestWI2ListByAssigneeFilter() {
 	assert.True(s.T(), strings.Contains(*list.Links.First, "filter[assignee]"))
 }
 
+func (s *WorkItem2Suite) TestWI2ListByNoAssigneeFilter() {
+	// given
+	userType := "identities"
+	newUser := createOneRandomUserIdentity(s.svc.Context, s.DB)
+	newUserID := newUser.ID.String()
+	c := minimumRequiredCreatePayload()
+	c.Data.Attributes[workitem.SystemTitle] = "Title"
+	c.Data.Attributes[workitem.SystemState] = workitem.SystemStateNew
+	c.Data.Relationships.BaseType = newRelationBaseType(space.SystemSpace, workitem.SystemBug)
+	c.Data.Relationships.Assignees = &app.RelationGenericList{
+		Data: []*app.GenericData{
+			{
+				Type: &userType,
```

Comment:
`idnType` is a constant, but I have changed the initialization of `userType` with the value of `APIStringTypeUser` constant.